src/main/java/uk/gov/hmcts/reform/sscs/personalisation/Personalisation.java

```diff
@@ -151,12 +50,10 @@
         APPEAL_RECEIVED_NOTIFICATION, SYA_APPEAL_CREATED_NOTIFICATION, EVIDENCE_RECEIVED_NOTIFICATION, NON_COMPLIANT_NOTIFICATION, VALID_APPEAL_CREATED);
 
     private static final String CRLF = String.format("%c%c", (char) 0x0D, (char) 0x0A);
-
-    private boolean sendSmsSubscriptionConfirmation;
-
+    private static final String READY_TO_LIST = "readyToList";
```

Comment:
This doesn't need to be a local variable - you can just use the State.READY_TO_LIST value which gives you the same thing. We use this in most other places so would be good if this could be changed.

Reply:
Changed